src/codegen/encode_methods.jl

```diff
@@ -30,15 +30,16 @@ function field_encode_expr(@nospecialize(f::FieldType), ctx::Context)
         !isempty(encoding_val_type) && (encoding_val_type = ", Val{$encoding_val_type}")
         # TODO: do we want to allow unpacked representation? Docs say that parsers must always handle both cases
         # and since packed is strictly more efficient, currently we don't allow that.
-        # is_packed = parse(Bool, get(f.options, "packed", "false"))
-        # if is_packed
+        is_packed = parse(Bool, get(f.options, "packed", "false"))
```

Comment:
I think we want `get(f.options, "packed", "true")` So people really have to make an effort to get the unpacked representation, i.e. they have to write `[packed=false]` in their proto files.

Reply:
Right. Sorry. In my case i needed it other way around and forgot about it.